ui/dashboards/src/components/Panel/PanelHeader.tsx

```diff
@@ -96,14 +97,25 @@ export function PanelHeader({ id, title, description, editHandlers, isHovered, s
           {title}
         </Typography>
       }
-      action={action}
+      action={
+        <HeaderActionWrapper direction="row" spacing={0.25} alignItems="center">
+          {actions}
+        </HeaderActionWrapper>
+      }
       sx={combineSx(
         (theme) => ({
           padding: theme.spacing(1),
           borderBottom: `solid 1px ${theme.palette.divider}`,
           '.MuiCardHeader-content': {
             overflow: 'hidden',
           },
+          '.MuiCardHeader-action': {
+            // Overriding the negative margins from MUI's defaults, so we
+            // can vertically center the icons. Moving these values to a wrapper
+            // inside the action in `HeaderActionWrapper` below.
+            // https://github.com/mui/material-ui/blob/master/packages/mui-material/src/CardHeader/CardHeader.js#L56-L58
```

Comment:
MUI's defaults seem to optimize for the icons being in the upper right corner. We want them to be vertically centered.